wbooks/src/app/screens/DetailBook/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const DETAILBOOK_CONST = {
+  ADD_WISH: 'Add to whislist',
+  CHECK: '✔',
+  EMPTY: '',
+  ANIMATE_DURATION: 2000,
+  USE_NATIVE_DRIVE: false
```

Comment:
animate duration should be here, just a const, and native driver shouldn't exist at all